### ISSUE TITLE
Easier Social Sharing

### ISSUE THREAD

**Issue by jasonphillips** [encyclical/laudato-si#3]
I put up an initial attempt at very minimalist social link sharing that appears at the top when needed. So far only Twitter, though I'll add a Facebook link at the very least. I'd also like to add URL shortening through bit.ly or another service.

**Developed in branch:** 
https://github.com/jasonphillips/laudato-si/tree/link-share

**Preview on my copy:** 
http://jasonphillips.github.io/laudato-si/

Suggestions? I'll move towards a pull request once at least covering the above items and anything else that is deemed necessary.

**Comment by jasonphillips** [comment]
Have a look; added bitly integration and a Facebook link. Simple implementation, hopefully pretty compatible but let me know.

**Comment by englishm** [comment]
I like this. There is an issue with the popup for copying the link though - it appears again for me after I OK or Cancel the dialog. (Chrome on OS X 10.10)

**Comment by englishm** [comment]
Actually, that was earlier today. I just tried again now and it's no longer happening.

**Comment by jasonphillips** [comment]
Yes, sorry about that, I corrected my rebinding mistake earlier which was causing the duplication.

It could likely use some further styling and tweaks, as it was a bit hastily put together in the few moments I could steal today.
